src/MicroflowTimer/widget/MicroflowTimer.js

```diff
@@ -161,6 +163,14 @@ define([
             }
         },
 
+        _executeEvent: function() {
+            if(this.callEvent == "callMicroflow") {
```

Comment:
use strict comparison operators change to `if(this.callEvent === "callMicroflow") {`

Reply:
done